packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -443,39 +444,41 @@ export class SynapseTable extends React.Component<
 
     /* min height ensure if no rows are selected that a dropdown menu is still accessible */
     return (
-      <div className="SynapseTable SRC-overflowAuto" data-testid="SynapseTable">
-        <table
-          ref={node => (this.tableElement = node)}
-          className="table table-striped table-condensed"
+      <>
+        <div
+          className="SynapseTable SRC-overflowAuto"
+          data-testid="SynapseTable"
         >
-          <thead className="SRC_bordered">
-            <tr>
-              {this.createTableHeader(
+          <table ref={node => (this.tableElement = node)}>
+            <thead>
+              <tr>
+                {this.createTableHeader(
+                  headers,
+                  columnModels,
+                  facets,
+                  isShowingAccessColumn,
+                  isShowingDirectDownloadColumn,
+                  isShowingAddToV2DownloadListColumn,
+                  isRowSelectionVisible,
+                )}
+              </tr>
+            </thead>
+            <tbody>
+              {this.createTableRows(
+                rows,
                 headers,
-                columnModels,
-                facets,
                 isShowingAccessColumn,
                 isShowingDirectDownloadColumn,
                 isShowingAddToV2DownloadListColumn,
                 isRowSelectionVisible,
               )}
-            </tr>
-          </thead>
-          <tbody>
-            {this.createTableRows(
-              rows,
-              headers,
-              isShowingAccessColumn,
-              isShowingDirectDownloadColumn,
-              isShowingAddToV2DownloadListColumn,
-              isRowSelectionVisible,
-            )}
-          </tbody>
-        </table>
-        <div className="bootstrap-4-backport" style={{ textAlign: 'right' }}>
-          <TablePagination />
+            </tbody>
+          </table>
         </div>
-      </div>
+        <Box sx={{ mt: 2, textAlign: 'right' }}>
+          <TablePagination />
+        </Box>
```

Comment:
Moved table pagination out of the SynapseTable container, so now it stays in place if you scroll the table in the `x` direction

Reply:
https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/9606af5a-ee72-4d98-b9b5-39a707585499